src/actions/window.js

```diff
@@ -40,3 +40,13 @@ export function addWindow(options) {
 export function removeWindow(windowId) {
   return { type: ActionTypes.REMOVE_WINDOW, windowId };
 }
+
+/**
+ * toggleWindowSideBar - action creator
+ *
+ * @param  {String} windowId
+ * @memberof ActionCreators
+ */
+export function toggleWindowSideBar(windowId) {
+  return { type: ActionTypes.TOGGLE_WINDOW_SIDE_BAR, windowId };
```

Comment:
I wrestled w/ how to handle this in the state.  This is the simplest implementation that I could think of, but wasn't sure if I should go deeper w/o knowing how we'll want to model this in the state down the road (e.g. I could set this as its own object w/i the window for future extension).

Happy to go down other routes w/ how to handle this in the state now or we can kick that down the road.